gvm/protocols/gmpv7/types.py

```diff
@@ -563,6 +567,7 @@ class ScannerType(Enum):
     OPENVAS_SCANNER_TYPE = "2"
     CVE_SCANNER_TYPE = "3"
     GMP_SCANNER_TYPE = "4"  # formerly slave scanner
+    GREENBONE_SENSOR_SCANNER_TYPE = "5"
```

Comment:
I am really sure this is only valid for GMPv9. You need to create a copy of the `ScannerType` enum in `gvm/protocols/gmpv9/types.py` and only add the `GREENBONE_SENSOR_SCANNER_TYPE` to this new enum.